x/poe/keeper/test_common.go

```diff
@@ -222,6 +225,12 @@ func createTestInput(
 	stakingAdapter := stakingadapter.NewStakingAdapter(nil, nil)
 	twasmSubspace := paramsKeeper.Subspace(twasmtypes.DefaultParamspace)
 
+	consensusParamsUpdater := baseapp.NewBaseApp("testApp", log.NewNopLogger(), db, nil)
+	// set the BaseApp's parameter store
+	consensusParamsUpdater.SetParamStore(paramsKeeper.Subspace(baseapp.Paramspace).WithKeyTable(paramskeeper.ConsensusParamsKeyTable()))
+
+	consensusParamsUpdater.StoreConsensusParams(ctx, app.DefaultConsensusParams)
```

Comment:
good idea to use the baseapp here. 👍

Reply:
Yes, it's a little "dirty" in that I'm returning it in `testKeepers`, with the keepers. But it's worth it, for simplicity.